2.3.4/bullseye/trivyignore

```diff
@@ -1,4 +1,2 @@
-# Ignore these CVEs only for the dev image.
-# The release image would pass the CVE scan
-CVE-2022-38054
+# Airflow vulnerability fixed in 2.4.0
```

Comment:
```suggestion
# Airflow vulnerability with backported fix
```

Knowing it was fixed in 2.4.0 isn't as useful as knowning we've already backported the fix (meaning we can safely ignore it here).